### PR TITLE
[New] Ruby on Rails with One-Click Apps

### PR DESCRIPTION
**Pull request by hzoppetti**
draft guide

### CODE REVIEW

**Comment by nmelehan** [comment]
Deploy preview for *nostalgic-ptolemy-b01ab8* ready!

Built with commit 3b14bbbea51524a66d050cd5922b3e365324b76c

https://deploy-preview-3188--nostalgic-ptolemy-b01ab8.netlify.com

File: docs/platform/one-click/deploying-ruby-on-rails-with-one-click-apps/index.md [hunk 1]
```diff
@@ -0,0 +1,160 @@
...
+
+1.  [SSH into your Linode](/docs/getting-started/#connect-to-your-linode-via-ssh) and [create a limited user account](/docs/security/securing-your-server/#add-a-limited-user-account).
+
+1.  Log out and log back in as your limited user account.
+
+1.  Update your server:
+
+        sudo apt-get update && apt-get upgrade
+
+1.  Ruby comes with some pre-made scripts to get you started. One of these is a blog, to start this, use the following command:
```
Comment (andystevensname) [inline docs/platform/one-click/deploying-ruby-on-rails-with-one-click-apps/index.md]: Split "One of these is a blog. To begin with the blog example,..." into two sentences.

File: docs/platform/one-click/deploying-ruby-on-rails-with-one-click-apps/index.md [hunk 2]
```diff
@@ -0,0 +1,160 @@
...
+create      app/assets/stylesheets/welcome.scss
+{{</ output >}}
+
+1.  With the text editor of your choice, edit the file `app/views/welcome/index.html.erb` and replace the contents with the following:
+
+    {{< file "app/views/welcome/index.html.erb" html >}}
+<h1>Hello, World! This is Ruby on Rails!</h1>
+{{</ file >}}
+
+1.  Tell Rails where to find the document root. Edit the file `config/routes.rb`, find and uncomment the line root line as shown:
```
Comment (andystevensname) [inline docs/platform/one-click/deploying-ruby-on-rails-with-one-click-apps/index.md]: "...line defining the root". Remove second "line".

File: docs/platform/one-click/deploying-ruby-on-rails-with-one-click-apps/index.md (hunk 1 above)
Comment (hzoppetti) [reply docs/platform/one-click/deploying-ruby-on-rails-with-one-click-apps/index.md]: updated

File: docs/platform/one-click/deploying-ruby-on-rails-with-one-click-apps/index.md (hunk 2 above)
Comment (hzoppetti) [reply docs/platform/one-click/deploying-ruby-on-rails-with-one-click-apps/index.md]: removed

**Pull request merged**